App.js

```diff
@@ -1,10 +1,45 @@
-import React from 'react';
-import { View } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import { View, Text, Dimensions } from 'react-native';
+import Spaceship from './src/components/Spaceship';
+import AxisPad from 'react-native-axis-pad';
+import { styles } from './src/styles';
 
+console.disableYellowBox = true;
 
 export default function App() {
+  const [left, setLeft] = useState(40);
+  const [cursorX, setCursorX] = useState(0);
+  const maxLeft = Dimensions.get('window').width - 36;
+
+  var intervalID
+  useEffect(() => {
+    if (cursorX != 0) {
+      intervalID = setInterval(() => {
+        setLeft(left => left > maxLeft ? maxLeft : left < 0 ? 0 : left += (5 * cursorX));
+      }, 10);
+    }
+    return () => clearInterval(intervalID);
+
+  }, [cursorX]);
+
   return (
-    <View></View>
+    <View style={styles.container}>
+      <Text>{left}</Text>
+      <AxisPad
+        wrapperStyle={{ position: 'absolute', bottom: 40 }}
```

Comment:
```suggestion
        wrapperStyle={{ position: 'absolute', bottom: 0, left: 150 }}
```
Subindo um pouco a posição da nave para {bottom: 100} fica perfeito! :D